### DISCUSSION TITLE
Customize dropdown menu

### DISCUSSION THREAD

**Discussion by qpwo** [rxhanson/Rectangle#447] [Ideas]
Would be nice if you could change the actions listed in the dropdown from clicking the icon in the menu bar. This is challenging UX to put in the preferences menu, so you could give the user a plaintext file that they can edit.

**Comment by rxhanson** [comment]
Now that the preferences can be exported/imported as json, this would be a nice fit as something configurable within that file. 

I don't have the bandwidth to add this in at the moment, but would definitely consider a PR for it.

**Comment by g3rhard** [comment]
Fully agree, don't use all of the features and have favourite sets of window positions, so prefer to have less options in drop-down menu.
Some of them even don't use (quarters, sixths, edges).
![image](https://github.com/user-attachments/assets/02b24014-d8b1-41e3-917e-63dd18310178)

**Reply by rxhanson** [reply]
I still would be fine merging a good pull request that does this without using the UI.

As an aside this feature is in the UI in [Rectangle Pro](https://rectangleapp.com/pro) (paid), in the Menu & Icon tab of the Settings window.
